.update-copyright.conf

```diff
@@ -0,0 +1,17 @@
+[project]
+name: lesson-template
```

Comment:
On Sun, Mar 15, 2015 at 07:53:33PM -0700, Greg Wilson wrote:
> > @@ -0,0 +1,17 @@
> > +[project]
> > +name: lesson-template
> 
> Can `update-copyright.py` infer the name of the repo by looking at
> its Git origin remote instead?

Hmm, maybe.  I'll have to check and see how Mercurial handles remotes.
And I think detecting the name of the upstream branch in Git can be a
bit tricky (it's not always ‘origin’), but I may just be
mis-remembering.  I'll look into it and report back.